app-header/app-header.html

```diff
@@ -444,7 +444,7 @@
        * @return {boolean}
        */
       isOnScreen: function() {
-        return this._height !== 0 && (this._top === 0 || this._top < this._height);
+        return this._height !== 0 && (this.fixed || this._top === 0 || this._top < this._height);
```

Comment:
Why is this needed? Shouldn't `this._top` be less than `this._height` if its fixed (and shouldn't `this._top === 0` not be needed as well)?